docs/spectral_index.md

```diff
@@ -236,6 +236,26 @@ Index range: -∞, ∞
     - hsi         - Hyperspectral image object, an instance of the `Spectral_data` class in plantcv (read in using [pcv.readimage](read_image.md) with `mode='envi'`)
     - distance    - Amount of flexibility (in nanometers) regarding the bands used to calculate an index.
 
+### NDCI
+
+Calculates the Normalized Difference Chlorophyll Edge index using reflectance values ([Mishra & Mishra, 2012](#references)):
```

Comment:
There is a link to the references but the reference was not added to the reference list